### PR TITLE
Add doc - Add application/user creation workflow

### PR DESCRIPTION
**Pull request by isuruuy429**
This PR includes the doc
- Adding application and user creation workflow
- A few typos in the existing content.

### CODE REVIEW

**Comment by sameeragunarathne** [comment]
Can you reorganize this under a topic such as "Advance Topics". Also resolve the current conflicts

File: en/mkdocs.yml [hunk 1]
```diff
@@ -113,9 +113,13 @@ nav:
     - EMR Connectivity:
       - EMR Systems Overview: emr-connectivity/guides/emr-systems-overview.md
       - Connecting with Epic EMR systems: emr-connectivity/guides/emr-connector.md
-    - Securing a Health APIs:
+    - Securing Health APIs:
       - SMART on FHIR: secure-health-apis/guides/smart-on-fhir.md
-      - Retrive Patient ID: secure-health-apis/guides/getting-patientid.md
+      - Configure SMART App Launch: secure-health-apis/guides/getting-patientid.md
+    - Advance Topics:
```
Comment (sameeragunarathne) [inline en/mkdocs.yml]:
```suggestion
    - Advanced Topics:
```

File: en/docs/advance-topics/guides/enable-workflow.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+Healthcare accelerator provides the following capabilities. 
+
+- View the user details in a user creation request. 
+- Option to approve/reject user creation request. 
+- View the application details in a application creation request. 
+- Option to approve/reject application creation request. 
+
+## How to enable the User/Application creation workflow 
+1. Follow the [Installation Steps](../../install-and-setup/manual.md) to install the Healthcare Accelerator.
```
Comment (sameeragunarathne) [inline en/docs/advance-topics/guides/enable-workflow.md]: Healthcare APIM Accelerator

File: en/docs/advance-topics/guides/enable-workflow.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+Healthcare accelerator provides the following capabilities. 
+
+- View the user details in a user creation request. 
+- Option to approve/reject user creation request. 
+- View the application details in a application creation request. 
+- Option to approve/reject application creation request. 
+
+## How to enable the User/Application creation workflow 
+1. Follow the [Installation Steps](../../install-and-setup/manual.md) to install the Healthcare Accelerator. 
+2. After extracting WSO2 OH APIM Accelerator to `<WSO2_APIM_HOME>`, navigate to `<WSO2_OH_APIM_ACC_HOME>`/conf/config.toml file, and enable the following configuration.
```
Comment (sameeragunarathne) [inline en/docs/advance-topics/guides/enable-workflow.md]:
```suggestion
2. After extracting WSO2 Healthcare APIM Accelerator to `<WSO2_APIM_HOME>`, navigate to `<WSO2_OH_APIM_ACC_HOME>`/conf/config.toml file, and enable the following configuration. 
```

Comment (isuruuy429) [reply en/docs/advance-topics/guides/enable-workflow.md]: Fixed in https://github.com/wso2/open-healthcare-docs/pull/45/commits/bc3c197716619abfbc8849f455e744b9f0f18df5

File: en/docs/advance-topics/guides/enable-workflow.md (hunk 2 above)
Comment (isuruuy429) [reply en/docs/advance-topics/guides/enable-workflow.md]: Fixed in https://github.com/wso2/open-healthcare-docs/pull/45/commits/bc3c197716619abfbc8849f455e744b9f0f18df5

File: en/mkdocs.yml (hunk 1 above)
Comment (isuruuy429) [reply en/mkdocs.yml]: Fixed in https://github.com/wso2/open-healthcare-docs/pull/45/commits/bc3c197716619abfbc8849f455e744b9f0f18df5

File: en/mkdocs.yml [hunk 4]
```diff
@@ -113,9 +113,13 @@ nav:
     - EMR Connectivity:
       - EMR Systems Overview: emr-connectivity/guides/emr-systems-overview.md
       - Connecting with Epic EMR systems: emr-connectivity/guides/emr-connector.md
-    - Securing a Health APIs:
+    - Securing Health APIs:
       - SMART on FHIR: secure-health-apis/guides/smart-on-fhir.md
-      - Retrive Patient ID: secure-health-apis/guides/getting-patientid.md
+      - Configure SMART App Launch: secure-health-apis/guides/getting-patientid.md
+    - Advanced Topics:
+      - Enable a workflow: advance-topics/guides/enable-workflow.md
```
Comment (sameeragunarathne) [inline en/mkdocs.yml:120]: This should be "Enable application creation workflow"?

Comment (isuruuy429) [reply en/mkdocs.yml:120]: Fixed in https://github.com/wso2/open-healthcare-docs/pull/45/commits/36aabfcfb4951987c81145c2be88e9ee45344c9b

**Review by sameeragunarathne — APPROVED** [review]

**Pull request merged**